packages/components/src/internal/components/search/SearchResultCard.tsx

```diff
@@ -20,53 +20,59 @@ import { SVGIcon } from '../base/SVGIcon';
-        if (isTopResult) incrementClientSideMetricCount(productId + 'Search', 'firstResultClicked');
-        incrementClientSideMetricCount(productId + 'Search', 'resultClicked');
-    }, [isTopResult, productId]);
+        const onClick = useCallback(() => {
+            if (isTopResult) incrementClientSideMetricCount(productId + 'Search', 'firstResultClicked');
+            incrementClientSideMetricCount(productId + 'Search', 'resultClicked');
+        }, [isTopResult, productId]);
 
-    const textParts = [];
-    if (category) textParts.push(category);
-    if (typeName) textParts.push(typeName);
-    return (
-        <a href={url} onClick={onClick}>
-            <div className="row search-result__card-container">
-                <div className="hidden-xs search-result__card-icon__container">
-                    {iconUrl && <img className="search-result__card-icon" src={iconUrl} alt={altText} />}
-                    {!iconUrl && (
-                        <SVGIcon
-                            iconDir={iconDir}
-                            iconSrc={iconSrc}
-                            alt={altText}
-                            className="search-result__card-icon"
-                        />
-                    )}
-                </div>
-                <div className="col-sm-12">
-                    <div className="col-sm-12 search-result__title">
-                        <h4>{title}</h4>
-                        {textParts.length > 0 && (
-                            <div className="margin-left status-pill muted">{textParts.join(': ')}</div>
+        const textParts = [];
+        if (category) textParts.push(category);
+        if (typeName) textParts.push(typeName);
+        return (
+            <a href={url} onClick={onClick}>
+                <div className="row search-result__card-container">
+                    <div className="hidden-xs search-result__card-icon__container">
+                        {iconUrl && <img className="search-result__card-icon" src={iconUrl} alt={altText} />}
+                        {!iconUrl && (
+                            <SVGIcon
+                                iconDir={iconDir}
+                                iconSrc={iconSrc}
+                                alt={altText}
+                                className="search-result__card-icon"
+                            />
                         )}
                     </div>
-                    <div className="search-result__summary">{summaryText}</div>
+                    <div className="col-sm-12">
+                        <div className="col-sm-12 search-result__title">
+                            <h4>{title}</h4>
+                            {textParts.length > 0 && (
+                                <div className="margin-left status-pill muted">{textParts.join(': ')}</div>
+                            )}
+                            {archived && <Alert className="folder-field_archived-tag">Archived</Alert>}
```

Comment:
I know it's small, but since we use this in a couple of places, I wonder if a small component, to assure consistency, would be a good idea.

Reply:
done